icu4c/source/test/intltest/itrbnf.cpp

```diff
@@ -1936,7 +1939,7 @@ IntlTestRBNF::TestAllLocales()
                             + UnicodeString(" -> ") + str + UnicodeString(" -> ") + num.getDouble());
                     }
                 }
-                if (!quick && !logKnownIssue("9503") ) {
```

Comment:
👍

Reply:
@grhoten, let me put here the question I put on the CR: 
- Can we remove the _!quick_ guard as well?
By skipping the redundancy, the test now executes in ~2-3s (against 25~30s with redundancy).

Reply:
I have no objections with removing the quick check.